velox/core/PlanNode.h

```diff
@@ -1669,7 +1669,8 @@ class MergeJoinNode : public AbstractJoinNode {
 ///
 /// Nested loop join supports both equal and non-equal joins. Expressions
 /// specified in joinCondition are evaluated on every combination of left/right
-/// tuple, to emit result.
+/// tuple, to emit result. Results are emitted following the same input order of
```

Comment:
It would be nice to also update documentation in https://facebookincubator.github.io/velox/develop/operators.html#nestedloopjoinnode

Should we clarify that the order is preserved only within a single thread of execution?